setup.cfg

```diff
@@ -2,8 +2,7 @@
 # References:
 # https://flake8.readthedocs.io/en/latest/user/configuration.html
 # https://flake8.readthedocs.io/en/latest/user/error-codes.html
-
-# Note: there cannot be spaces after comma's here
+# https://pycodestyle.pycqa.org/en/latest/intro.html#error-codes
```

Comment:
The existing link, `https://flake8.readthedocs.io/en/latest/user/error-codes.html`, only covers error codes starting with `F`. This additional link contains error codes starting with `E` and `W` as well.

The existing link directs readers to the new link, as seen below.
> Users should also reference pycodestyle’s list of error codes.